src/fa/game_session.py

```diff
@@ -177,13 +178,41 @@ def _turn_state_changed(self, val):
 
     def _launched(self):
         self._logger.info("Game has started")
+        with open(util.LOG_FILE_GAME, 'r') as src, open(util.LOG_FILE_GAME+'.full.log', 'a+') as dst: dst.write(src.read())
```

Comment:
I'm not sure if you shouldn't clear the LOG_FILE_GAME here. It's hard to tell when the code isn't split up into functions.

Reply:
code split up into function

Reply:
code split up into function